packages/pluggableWidgets/switch-native/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "switch-native",
   "widgetName": "Switch",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
I suppose since this widget is yet to be released, the version can stay at 1.0.0